python/tskit/tables.py

```diff
@@ -1665,27 +1665,45 @@ def simplify(
+        Returns an :class:`EdgeTable` instance describing a subset of the genealogical
+        relationships between the nodes in``samples`` and ``ancestors``.
+
         Each row ``parent, child, left, right`` in the output table indicates that
-        ``child`` (which is either in ``samples`` or in ``ancestors``) has inherited the
-        segment ``[left, right)`` from ``parent`` (which is in ``ancestors``) more
-        recently than from any other node in ``ancestors``. Every segment of the
-        genome on which each of ``samples`` has inherited from any of ``ancestors``
-        is described in this way, as well as any segment of the genomes of ``ancestors``
-        that has been inherited by any of ``samples``.  The node IDs in ``parent`` and
-        ``child`` refer to the node table in the original tree sequence. In other words,
-        the tree constructed at a position with the output edge table has the property
-        that the parent of any node in ``samples``, if there is one, is the most recent
-        ancestor of that node at that position out of all possible ancestors in
-        ``ancestors``, and the same is true for any node in ``ancestors`` that is
-        ancestral to any of ``samples``.
-
-        The supplied nodes must be a subset of the node IDs in the tree sequence.
-        Other than this, the procedure is flexible; overlap between the lists of samples
-        and ancestors is permitted, and the nodes need not be ordered by time.
-        If none of the nodes in `ancestors` are ancestral to `samples` anywhere in
-        the tree sequence, an empty table will be returned.
+        ``child`` has inherited the segment ``[left, right)`` from ``parent`` more
+        recently than from any other node in these lists.
+
+        In particular, suppose ``samples`` is a list of nodes such that ``time`` is 0
+        for each node, and ``ancestors`` is a list of nodes such that ``time`` is
+        greater than 0.0 for each node. Then each row of the output table will show
+        an interval ``[left, right)`` over which a node in ``samples`` has inherited
+        most recently from a node in ``ancestors``, or an interval over which one of
+        these ``ancestors`` has inherited most recently from another node in
+        ``ancestors``.
+
+        The following table shows which ``parent->child`` pairs will be shown in the
+        output of ``map_ancestors``.
+
+        | Type of relationship     | Shown in output of ``map_ancestors``            |
+        | ------------------------ | ----------------------------------------------- |
+        | ``ancestor->sample``     | Always                                          |
+        | ``ancestor1->ancestor2`` | Only if ``ancestor2`` has a relevant descendant |
+        | ``sample1->sample2``     | Always                                          |
+        | ``sample->ancestor``     | Only if ``ancestor`` has a relevant descendant  |
```

Comment:
vote to change "relevant descendant" to "descendant in samples or ancestors"

Reply:
this isn't quite true though - it will only keep a descendant in `ancestors` if that node is also parent to another edge in the table.

Reply:
o right. =) so, shouldn't it say "has a descendant in samples"? or, "is ancestral to a sample"?

Reply:
hmm, on its own that wording is also a bit confusing I think, because it might not be immediately ancestral to a sample - it could be ancestral to an ancestor that is ancestral to an ancestor... that is ancestral to a sample, for example.

I've added an extra sentence to say what I mean by 'relevant descendant', but I can also add something like 'has a descendant in samples on the given interval'